### PR TITLE
Task#399 update e.g address

### PR DESCRIPTION
**Pull request by NemTechCN**
update e.g address

### CODE REVIEW

File: src/language/en-US.ts [hunk 1]
```diff
@@ -195,7 +195,7 @@ export const en_US = {
     send: 'Send',
     transfer_record: 'Transfer records',
     transfer_sent: 'Transfer sent',
-    receive_address_or_alias: 'Receive address or @+alias（@nem.xem）',
+    receive_address_or_alias: 'Enter address or @+alias（e.g., @nem.foundation）',
```
Comment (decentraliser) [inline src/language/en-US.ts]: we don't use "`@`"

File: src/common/vue/collection-record/CollectionRecordTs.ts [hunk 2]
```diff
@@ -88,6 +88,7 @@ export class CollectionRecordTs extends Vue {
     }
     
     get displayedDate(): string {
+        //@ts-ignore
```
Comment (decentraliser) [inline src/common/vue/collection-record/CollectionRecordTs.ts]: should not be necessary

maybe `:Date` as a return type would help?

File: src/language/zh-CN.ts [hunk 3]
```diff
@@ -181,7 +182,7 @@ export const zh_CN = {
     send: '发送',
     transfer_record: '转账记录',
     transfer_sent: '付款记录',
-    receive_address_or_alias: '直接输入接收地址或@+别名（@nem.xem）',
+    receive_address_or_alias: '直接输入接收地址或@+别名（例如：@nem.foundation）',
```
Comment (decentraliser) [inline src/language/zh-CN.ts]: remove "@"

File: src/views/service/mosaic/mosaic-function/mosaic-list/MosaicListTs.ts [hunk 4]
```diff
@@ -14,6 +14,7 @@ import {
     sortByMosaicTransferable
 } from "@/core/services"
 import Alias from '@/views/forms/alias/Alias.vue'
+import App from '../../../../../App.vue';
```
Comment (decentraliser) [inline src/views/service/mosaic/mosaic-function/mosaic-list/MosaicListTs.ts]: should not be here

File: src/language/en-US.ts (hunk 1 above)
Comment (NemTechCN) [reply src/language/en-US.ts]: fixed

File: src/common/vue/collection-record/CollectionRecordTs.ts (hunk 2 above)
Comment (NemTechCN) [reply src/common/vue/collection-record/CollectionRecordTs.ts]: fixed

File: src/language/zh-CN.ts (hunk 3 above)
Comment (NemTechCN) [reply src/language/zh-CN.ts]: fixed

File: src/views/service/mosaic/mosaic-function/mosaic-list/MosaicListTs.ts (hunk 4 above)
Comment (NemTechCN) [reply src/views/service/mosaic/mosaic-function/mosaic-list/MosaicListTs.ts]: fixed, this pr should only update i18l file

**Pull request merged**